datahub-web-react/src/app/entity/chart/ChartEntity.tsx

```diff
@@ -209,4 +209,15 @@ export class ChartEntity implements Entity<Chart> {
             getOverrideProperties: this.getOverridePropertiesFromEntity,
         });
     };
+
+    supportedCapabilities = () => {
+        return new Set([
+            EntityCapabilityType.OWNERS,
+            EntityCapabilityType.GLOSSARY_TERMS,
+            EntityCapabilityType.TAGS,
+            EntityCapabilityType.DOMAINS,
+            EntityCapabilityType.DEPRECATION,
+            EntityCapabilityType.SOFT_DELETE,
```

Comment:
thinking out loud here: can we somehow unify capabilities & entity page layout some day so that we don't have to extend both places?

Reply:
Possibly - But I'm not sure the two are strictly equivalent... For example, the stats tab merges a few things already

Reply:
That is, entity capabilities not necessarily 1:1 with tabs